server/master/src/main/java/org/apache/accumulo/master/tableOps/CreateDir.java

```diff
@@ -39,6 +43,13 @@ public long isReady(long tid, Master environment) throws Exception {
   public Repo<Master> call(long tid, Master master) throws Exception {
     VolumeManager fs = master.getFileSystem();
     fs.mkdirs(new Path(tableInfo.dir));
```

Comment:
I noticed the new code checks the boolean returned from mkdir, however the existing code does not.